drafts/doc-10-steps/document/0000-preface.adoc

```diff
@@ -0,0 +1,9 @@
+<<<
+== Preface
+=== Confidentiality, Copyright, and Disclaimer ===
+This is a Customer-facing document between Red Hat, Inc. and {customer} (“Client”).
+
+Copyright (C) {docdate} Red Hat, Inc. All Rights Reserved. No part of the work covered by the copyright herein may be reproduced or used in any form or by any means – graphic, electronic, or mechanical, including photocopying, recording, taping, or information storage and retrieval systems – without permission in writing from Red Hat except as is required to share this information as provided with the aforementioned confidential parties.
```

Comment:
Would it make sense to adjust the [license](https://github.com/theforeman/foreman-documentation/blob/master/LICENSE) before merging to master?

Reply:
We should probably remove that file (0000-preface.adoc) completely, assume the same CC license as the main repo. 

Regarding the images, we'll want to eventually replace those images completely with something that has community branding to begin with. Maybe remove them too for now?

Reply:
Good. I will start the GHA again; once this is green, I will approve this PR.